doc/crypt.3

```diff
@@ -433,14 +452,18 @@ l l l.
 Interface	Attribute	Value
 T{
 .Nm crypt
-T}	Thread safety	MT-Unsafe race:crypt
+T}	Thread safety	MT-Safe*
 T{
 .Nm crypt_r ,
 .Nm crypt_rn ,
 .Nm crypt_ra
 T}	Thread safety	MT-Safe
 .TE
-.sp
+.Pp
+\&* In this implementation.
+Do NOT rely on this to be portable behaviour.
```

Comment:
libxcrypt currently mostly uses the `behavior` spelling (not `behaviour`), so maybe it's best to use that consistently.